pwndbg/commands/radare2.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import subprocess
+
+import pwndbg.commands
+
+parser = argparse.ArgumentParser(description=".",
+                                 epilog="Example: radare2 -- -S -AA")
+parser.add_argument('--no-seek', action='store_true',
+                    help='Do not seek to current pc')
+parser.add_argument('arguments', nargs='*', type=str,
+                    help='Arguments to pass to radare2')
+
+@pwndbg.commands.ArgparsedCommand(parser)
+def radare2(arguments, no_seek=False):
```

Comment:
of cause, why not... i though about the same... but decided to use radare2 as r2 is a symlink to it.... but i like r2 also more.